### PR TITLE
Added release notes for 1.8.1 version

### PR DESCRIPTION
**Pull request by ifbyol**
This release includes a couple of fixes. One in the backend and another one in the CLI

### CODE REVIEW

**Review by pchico83 — APPROVED** [review]

File: src/content/self-hosted/install/releases.mdx [hunk 1]
```diff
@@ -5,6 +5,15 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.8.1
+23 May, 2023
+
+### Bugfixes
+
+- Fixed an issue deleting dev and preview environments when using custom okteto manifests. The destroy operation was not taken into account the manifest used during deployment and it wasn't executing the destroy commands defined in the manifest.
```
Comment (omnomagonz) [inline src/content/self-hosted/install/releases.mdx:13]:
```suggestion
- Fixed an issue affecting deletion of dev and preview environments when using custom Okteto manifests. The destroy operation did not evaluate the custom manifest used during deployment, which meant the destroy commands were not executed.
```

non-blocking suggestion

File: versioned_docs/version-1.8/self-hosted/install/releases.mdx [hunk 2]
```diff
@@ -5,6 +5,15 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.8.1
+23 May, 2023
+
+### Bugfixes
+
+- Fixed an issue deleting dev and preview environments when using custom okteto manifests. The destroy operation was not taken into account the manifest used during deployment and it wasn't executing the destroy commands defined in the manifest.
```
Comment (omnomagonz) [inline versioned_docs/version-1.8/self-hosted/install/releases.mdx:13]:
```suggestion
- Fixed an issue affecting deletion of dev and preview environments when using custom Okteto manifests. The destroy operation did not evaluate the custom manifest used during deployment, which meant the destroy commands were not executed.
```

non-blocking suggestion

File: versioned_docs/version-1.8/self-hosted/install/releases.mdx [hunk 3]
```diff
@@ -5,6 +5,15 @@ sidebar_label: Release Notes
...
+- Upgraded the Okteto CLI version to [2.15.3](https://github.com/okteto/okteto/releases/tag/2.15.3).
+  - Build optimization now takes into account environment variables used in build args to rebuild an image or not.
```
Comment (omnomagonz) [inline versioned_docs/version-1.8/self-hosted/install/releases.mdx:15]:
```suggestion
  - Build optimization now considers environment variables used in build args to determine when to rebuild an image.
```

non-blocking suggestion

**Comment by ifbyol** [comment]
Thanks for the feedback @omnomagonz ! Applied all of them!

**Comment by ifbyol** [comment]
@omnomagonz Could you give another pass to the review? I added a new entry for the bugfix section `Fixed a regression which was preventing the deploy dialog to load automatically the list of repositories accesible in GitHub for new users when the authentication provider was GitHub.`

Thanks in advance!

**Pull request merged**